pkgs/applications/audio/spotifyd/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, fetchFromGitHub, rustPlatform, pkgconfig, openssl
+, withALSA ? true, alsaLib ? null
+, withPulseAudio ? false, libpulseaudio ? null
+, withPortAudio ? false, portaudio ? null
+}:
+
+rustPlatform.buildRustPackage rec {
+  name = "spotifyd-${version}";
```

Comment:
Please replace with `pname = "spotifyd";` and `name` will be taken care of for you.

Reply:
Done.